marklogic-data-hub-central/ui/src/components/job-results-table-view/job-results-table-view.tsx

```diff
@@ -27,100 +30,94 @@ const JobResultsTableView = (props) => {
     setJobId("");
   };
 
+  const flowName = "Flow Name";
+  const user = "User";
+
   const {
     setMonitorSortOrder
   } = useContext(MonitorContext);
   let sorting = true;
   const columnOptionsLabel = {
-    user: "User",
-    flowName: "Flow Name",
+    user: user,
+    flowName: flowName,
+  };
+
+  const emptyValue = () => {
+    return <></>;
   };
 
   const MANDATORY_HEADERS = [
     {
-      text: "Job ID",
+      text: "Step Name",
       dataField: "jobId",
       visible: true,
-      width: 200,
+      //width: 320,
       sort: false,
       formatter: (jobId) => {
-        return <><a onClick={() => handleOpenJobResponse(jobId)}>{jobId}</a></>;
+        return <>
+          <HCTooltip text="Click the JOB ID to see the details" id="Click JOB ID to see the details" placement="right">
+            <a onClick={() => handleOpenJobResponse(jobId)}>{jobId}</a>
+          </HCTooltip>
+        </>;
       }
     },
-    {
-      text: "Step Name",
-      dataField: "stepName",
-      visible: true,
-      width: 200,
-      sort: true,
-    },
+    // {
```

Comment:
Any particular reason why we kept the commented out code here ? I see a lot of code commented out in different files in the PR.

Reply:
Yes @xnikhil08 in this case I´m keeping some code in this PR because the next one is about sorting this table, and I will use some of this. After that, I will clean all the remaining commented code.